src/webserver.ts

```diff
@@ -13,8 +18,51 @@ if (process.env.NODE_ENV !== "production") {
   );
 }
 
+//Whenever twitch sends a notification to your subscribed webhook topic
+//it will send it to this endpoint. You have to send back a 200
+//otherwise twitch will think you did not recieve the notification and spam you
+app.post(
+  "/webhooks/subscribe/:member_id",
+  async (req: Request, res: Response) => {
```

Comment:
Since this is an async function, it returns a promise. Express doesn't await it, so if the promise rejects then it will not reach error handler middleware (if and when it'll be implemented).
You can mitigate this by wrapping the async function declaration in something like this: https://github.com/soryy708/nodejs-app-quick-start/blob/master/src/api/routes/util/asyncWrapper.js